chroma-manager/tests/integration/existing_filesystem_configuration/utils/create_lustre_filesystem.py

```diff
@@ -259,20 +259,12 @@ def configure_target_device(self, target, target_type, fsname, mgs_nids,
 
         block_device = TestBlockDevice(device_type, device_path)
 
-        self.execute_simultaneous_commands(
-            block_device.install_packages_commands, targets.values(),
-            'install blockdevice packages')
-
-        self.execute_commands(block_device.prepare_device_commands,
-                              targets['primary_server'], 'prepare device')
+        self.execute_commands(block_device.reset_device_commands,
+                              targets['primary_server'], 'reset device')
 
         filesystem = TestFileSystem(block_device.preferred_fstype,
                                     block_device.device_path)
 
-        self.execute_simultaneous_commands(
-            filesystem.install_packages_commands, targets.values(),
-            'install filesystem packages')
-
```

Comment:
we aren't using the filesystem install_packages_commands for any filesystem type, we can reinstate this in the future if needed (which is unlikely for the moment)

Reply:
I see that 242a88762b22ea0478afedfdeaa857aac73b736a did re-instate these.  How were the packages getting installed before 242a88762b22ea0478afedfdeaa857aac73b736a, or was 242a88762b22ea0478afedfdeaa857aac73b736a added because it was determined that they were still needed?

Just trying to follow the path.